### ISSUE TITLE
[FEATURE REQUEST] Window preview on hover & minimize on click

### ISSUE THREAD

**Issue by astro-stan** [pop-os/cosmic-dock#21]
After using the COSMIC dock for a few days I noticed it does not support some of my favorite features from the dash-to-dock extension:

1. Ability to minimize windows. Basically a toggle. Click once - bring window up, click again - minimize it, click third time - bring it back up.
2. Ability to preview open windows on hover. It is quite tedious to have to click 4 times to get through to the PDF you want and if you miss - you have to start over. I realize there are other ways do this (alt + `) but I like to have the ability to do it from the dock as well.

**Comment by NathanaelA** [comment]
On your `#1`  Shift-Click will minimize.    (But you can use the same steps below to change shift-click to minimize & restore)

On your `#2`  You can use "dconf-editor" and then navigate to 'org.gnome.shell.extensions.dash-to-dock' and then you can see all the settings.    I personally set "click-action" to "minimize-or-previews".   This will do on a single "click":
1. Single app in background, focus it.
2. Multiple apps in background, show the previews so you can pick the app you want.
3.  App already focused, minimize it.

It isn't as good as on "hover", but I didn't see any hover code in the shell, so it would have to be added...

**Comment by astro-stan** [comment]
Thanks, that did the trick. It would still be nice to have a bit more flexibility in the settings though!

**Comment by VividLemon** [comment]
I agree, it is nice that this stuff is in the dconf-editor, but could be expanded to be included in the settings app
